source/hardware/controllers/stanton_djc_4.rst

```diff
@@ -56,174 +55,116 @@ The **Master** and **Mic Level** are hardware controls and interact directly wit
+6                         LOOP LENGTH X                                           Doubles beatloop size.
+7                         BEAT MULTIPLIER encoder                                 Turn to move the loop left or right by 1 beat per click.
+7                         BEAT MULTIPLIER button                                  Sets a loop using the current beatloop size long and enable the loop.
+7                         SHIFT + BEAT MULTIPLIER button                          Activates a rolling loop using the current beatloop size.
+========================  ======================================================  ===========================================================================================
+
+Sampler section
+~~~~~~~~~~~~~~~
+
+========================  ======================================================  ===========================================================================================
+No.                       Control                                                 Function
+========================  ======================================================  ===========================================================================================
+1                         :hwlabel:`SAMPLER 1-4` (left deck)                      Controls Samplers 1-4 (independent of deck selection)
+2                         :hwlabel:`SAMPLER 1-4` (right deck)                     Controls Samplers 5-8 (independent of deck selection)
+========================  ======================================================  ===========================================================================================
+
+
+Effect section
+~~~~~~~~~~~~~~
+
+========================  ======================================================  ===========================================================================================
+No.                       Control                                                 Function
+========================  ======================================================  ===========================================================================================
+1                         :hwlabel:`ON`                                           Toggle FX 1 for decks 1/3 (both on the left) and FX 2 for decks 2/4 (both on the right).
+2-5                       *Various*                                               See `Standard effects mapping <https://github.com/mixxxdj/mixxx/wiki/Standard-Effects-Mapping>`__.
+========================  ======================================================  ===========================================================================================
 
 Tweakables
-~~~~~~~~~~
+^^^^^^^^^^
 
-At the top of the file \`Stanton-DJC-4-scripts.js\` there are a few customizable options to change the default mapping.
+At the top of the file file:`Stanton-DJC-4-scripts.js` there are a few customizable options to change the default mapping.
```

Comment:
The wording for `autoShowFourDecks` below is strange. Can you change that to
`If a track gets loaded into deck 3 or 4, automatically show four decks in Mixxx`